### PR TITLE
Added integration test for getExistsStoreValueFlag in product model

### PR DESCRIPTION
**Pull request by ochnygosch**
### Description (*)

Add integration tests to demonstrate issue magento/magento2#26581

### Fixed Issues (if relevant)

1. magento/magento2#<issue_number>: Issue title

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php [hunk 1]
```diff
@@ -723,4 +723,32 @@ public function productWithBackordersDataProvider(): array
...
+
+    /**
+     * @magentoDataFixture Magento/Catalog/_files/products_different_store_values.php
+     * @magentoDbIsolation disabled
+     * @magentoAppIsolation enabled
+     * @magentoConfigFixture current_store catalog/frontend/flat_catalog_product 1
+     */
+    public function testExistsStoreValueFlagForMultipleProducts() {
+
+        $descriptionProudct = $this->productRepository->get('store_description');
```
Comment (edenduong) [inline dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php]: Please correct the typo: `$descriptionProduct`

File: dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php [hunk 2]
```diff
@@ -723,4 +723,32 @@ public function productWithBackordersDataProvider(): array
...
+        $nameProudct = $this->productRepository->get('store_name');
+
+        /** @var \Magento\Store\Model\StoreManagerInterface $storeManager */
+        $storeManager = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->get(\Magento\Store\Model\StoreManagerInterface::class);
```
Comment (edenduong) [inline dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php]: Please `use Magento\TestFramework\Helper\Bootstrap;` and `use Magento\Store\Model\StoreManagerInterface;` and replace it by alias

File: dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php [hunk 3]
```diff
@@ -723,4 +723,32 @@ public function productWithBackordersDataProvider(): array
...
+        $store = $storeManager->getStore('fixturestore');
+
+        $nameProudct->addAttributeUpdate('name','Overwritten Name',$store->getId());
+        $descriptionProudct->addAttributeUpdate('description', 'Overwritten Description', $store->getId());
+
+        $descriptionProudct = $this->productRepository->get('store_description',false, $store->getId(), true);
+        $nameProudct = $this->productRepository->get('store_name', false, $store->getId(), true);
```
Comment (edenduong) [inline dev/tests/integration/testsuite/Magento/Catalog/Model/ProductTest.php]: Please fix the typo `$nameProduct`

File: dev/tests/integration/testsuite/Magento/Catalog/_files/products_different_store_values_rollback.php [hunk 4]
```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+/** @var \Magento\Framework\Registry $registry */
+$registry = \Magento\TestFramework\Helper\Bootstrap::getObjectManager()->get(\Magento\Framework\Registry::class);
```
Comment (edenduong) [inline dev/tests/integration/testsuite/Magento/Catalog/_files/products_different_store_values_rollback.php]: Please declare `use` and use the alias

**Review by edenduong — CHANGES_REQUESTED** [review]
Thank you for your contribution. Please check my comment.

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by edenduong** [comment]
Hi @ochnygosch: Please fix the static test and the integration test. It should be green.

**Comment by edenduong** [comment]
Hello @ochnygosch,
After 30 days, I don't get any respond from you and the Automation Test still Fail so I will close this PR. It you want to continue, please reopen the PR.
Thank you !

**Comment by ochnygosch** [comment]
@magento give me test instance

**Pull request closed without merging**